webapp/stockstream/positions.py

```diff
@@ -5,7 +5,7 @@
 def compute_change_decimal(from_value, to_value):
     if from_value == 0 and to_value == 0:
         return 0
-    from_value = max(1, from_value)
+    from_value = 1 if from_value == 0 else from_value
```

Comment:
Change your mind?

Reply:
Yeah had to change this back so that it works for numbers between 0 and 1.